### PR TITLE
Nonlocal Potential BC with Schottky test fix

### PR DESCRIPTION
**Pull request by cticenhour**
In fixing up #16, I ran a check of the heavy tests, and discovered 1d_dc.non_local_potential_bc was broken. The problem seemed to be due to stricter input file check standards (a GetPot Expression for multiplication couldn't get past a type check for a parameter, and a parenthesis was left in as a typo). 

On fixing these the test still failed due to a "Sideset Distribution Factor" error (see below for a snippet from the exodiff output). However, the paraview output lines up very well with the original gold file, so I'm unsure what the problem still is. I figured I could open a PR for the obvious changes that needed to be made, and then we can decide if this is worth a re-gold of the old test or if something else is wrong. 

`Sideset Distribution Factors:
1d_dc.non_local_potential_bc:   --------- Time step 1, 1.0000000e-05 ~ 1.3525119e-08, rel diff:  9.98647e-01 (FAILED)`

### CODE REVIEW

File: tests/1d_dc/NonlocalPotentialBCWithSchottky.i [hunk 1]
```diff
@@ -47,7 +47,7 @@ area = 5.02e-7 # Formerly 3.14e-6
 
         trans_ss_check = 1
         ss_check_tol = 1E-15
-        ss_tmin = 3*${relaxTime}
+        ss_tmin = 3e-9 # or 3*${relaxTime}
```
Comment (lindsayad) [inline tests/1d_dc/NonlocalPotentialBCWithSchottky.i]: We can actually wait on idaholab/moose#12633 to go in and then we can use the new input file arithmetic parsing capability to update this line

Comment (cticenhour) [reply tests/1d_dc/NonlocalPotentialBCWithSchottky.i]: Sounds good!

**Comment by lindsayad** [comment]
I don't even know what sideset distribution factors are?! Do you @cticenhour?

**Comment by cticenhour** [comment]
So I was reading the 1994 [Exodus II manual](https://prod-ng.sandia.gov/techlib-noauth/access-control.cgi/1992/922137.pdf) the other day and it had this to say:

> This is an optional list of floating point factors associated with the nodes on a side set.
> These data may be used for uneven application of load or boundary conditions. Because
> distribution factors are assigned at the nodes, application codes that utilize these factors must
> read the side set node list. The distribution factors must be stored/accessed in the same order
> as the nodes in the side set node list; thus, the ordering conventions described [in section 3.11.4] apply.

This didn't make things much clearer to me.....are they some sort of weighting factors? If so, it seems like they aren't being set correctly somehow (I'm not sure how they're being set at all...).

**Comment by lindsayad** [comment]
>  If so, it seems like they aren't being set correctly somehow (I'm not sure how they're being set at all...).

Hmm well it seems like it was either incorrect before (when this gold file was created) and correct now, or correct before and incorrect now, or incorrect in different ways then and now :smile: Well I'm going to create an issue for resolving this heavy test

**Pull request merged**